.backend_env_azure.example

```diff
@@ -0,0 +1,26 @@
+SUPABASE_URL=<change-me>
+SUPABASE_SERVICE_KEY=<change-me>
+PG_DATABASE_URL=notimplementedyet
+OPENAI_API_KEY=<change-me>
+OPENAI_API_TYPE=azure
+OPENAI_API_BASE=https://<Your-Endpoint>.openai.azure.com/
+OPENAI_GPT_DEPLOYMENT_ID=<change-me>
+OPENAI_EMBEDDING_DEPLOYMENT_ID=<change-me>
+OPENAI_API_VERSION="2023-07-01-preview"
```

Comment:
> OPENAI_GPT_DEPLOYMENT_ID=<change-me>
> OPENAI_EMBEDDING_DEPLOYMENT_ID=<change-me>
> OPENAI_API_VERSION="2023-07-01-preview"

These 3 are Azure specific. Would it make sense to prefix them with `AZURE_`? 

AutoGPT uses these names:

> azure_api_type: azure
> azure_api_base: your-base-url-for-azure
> azure_api_version: api-version-for-azure
> azure_model_map:
>     fast_llm_deployment_id: gpt35-deployment-id-for-azure
>     smart_llm_deployment_id: gpt4-deployment-id-for-azure
>     embedding_model_deployment_id: embedding-deployment-id-for-azure

Reply:
Make sense. Rename those environments